fabric-ca-client/types/index.d.ts

```diff
@@ -2,177 +2,175 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-
-import { ICryptoSuite, User } from "fabric-common";
+import { ICryptoSuite, User } from 'fabric-common';
 
 declare class FabricCAServices {
-    constructor(url: string | FabricCAServices.IFabricCAService, tlsOptions?: FabricCAServices.TLSOptions, caName?: string, cryptoSuite?: ICryptoSuite);
-    getCaName(): string;
-    register(req: FabricCAServices.IRegisterRequest, registrar: User): Promise<string>;
-    enroll(req: FabricCAServices.IEnrollmentRequest): Promise<FabricCAServices.IEnrollResponse>;
-    reenroll(currentUser: User, attr_reqs: FabricCAServices.IAttributeRequest[]): Promise<FabricCAServices.IEnrollResponse>;
-    revoke(request: FabricCAServices.IRevokeRequest, registrar: User): Promise<any>;
-    generateCRL(request: FabricCAServices.IRestriction, registrar: User): Promise<any>;
-    newIdentityService(): FabricCAServices.IdentityService;
-    newAffiliationService(): FabricCAServices.AffiliationService;
-    toString(): string;
+	constructor(url: string | FabricCAServices.IFabricCAService, tlsOptions?: FabricCAServices.TLSOptions, caName?: string, cryptoSuite?: ICryptoSuite);
```

Comment:
`cryptoSuite` is optional here too, but mandatory in the code. It's a bit strange having a mandatory property following several optional ones too